src/components/SchedulePanel/components/RemoveHoursDialog/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import { gordonColors } from '../../../../theme';
+
+export default class RemoveHoursDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      removeOfficeHoursOpen: false,
+    };
+    this.handleRemoveOfficeHoursClose = this.handleRemoveOfficeHoursClose.bind(this);
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.onDialogSubmit();
+    console.log('Happened');
+    this.props.handleRemoveOfficeHoursClose();
+  };
+
+  handleRemoveOfficeHoursClose = () => {
```

Comment:
Do you mean to use this one or the one from the props? Because above, you use the one from the props.

Reply:
Change made